src/hooks/useCheckNickName.ts

```
@@ -0,0 +1,34 @@
+    const handleCheckNickName = async (nickName: string) => {
+        try {
+            setIsCheckingNickName(true);
+            const response = await checkNickName(nickName);
+            if (response.available) {
+                setNickNameMessage('사용 가능한 닉네임입니다.');
+                setNickNameAvailable(true);
+            } else {
+                setNickNameMessage('이미 사용 중인 닉네임입니다.');
+                setNickNameAvailable(false);
+            }
+        } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)